pipelines/vars/forklift_luna.yml

```diff
@@ -13,6 +13,7 @@ server_box:
         - "--puppet-server-max-active-instances 1"
         - "--puppet-server-jvm-min-heap-size 1G"
         - "--puppet-server-jvm-max-heap-size 1G"
+        - "--enable-foreman-cli-ansible"
```

Comment:
Did you want to run these new tests on luna as well?

Reply:
no, or maybe? probably! yes!